### PR TITLE
Fixes for v0.5

### PR DESCRIPTION
**Pull request by ranjanan**
This attempts to fix Graphs.jl for v0.5

### CODE REVIEW

File: src/common.jl [hunk 1]
```diff
@@ -19,13 +19,13 @@ vertex_index(v::KeyVertex) = v.index
 
 type ExVertex
     index::Int
-    label::UTF8String
+    label::String
```
Comment (KristofferC) [inline src/common.jl]: Maybe `Compat.UTF8String` if we still want this to be a concrete type.

File: src/dot.jl [hunk 2]
```diff
@@ -63,15 +63,15 @@ function to_dot(attrs::AttributeDict)
     if isempty(attrs)
         ""
     else
-        string("[",join(map(a -> to_dot(a[1], a[2]), attrs),","),"]")
+        string("[",join(map(a -> to_dot(a[1], a[2]), [attrs...]),","),"]")
```
Comment (tkelman) [inline src/dot.jl]: would collect work for this?

Comment (ranjanan) [inline src/dot.jl]: `collect(attrs)` works too.

File: .travis.yml [hunk 3]
```diff
@@ -3,8 +3,8 @@ os:
     - linux
     - osx
 julia:
-    - 0.3
```
Comment (tkelman) [inline .travis.yml]: don't delete this unless you also change the minimum julia version in REQUIRE

**Comment by ViralBShah** [comment]
Thanks @ranjanan

File: src/common.jl (hunk 1 above)
Comment (tkelman) [inline src/common.jl]: Yes. Otherwise this changes the API and performance on 0.4.

File: src/common.jl [hunk 4]
```diff
@@ -179,7 +179,7 @@ edge_property{T,V}(visitor::VectorEdgePropertyInspector{T}, e, g::AbstractGraph{
 edge_property_requirement{T, V}(visitor::AbstractEdgePropertyInspector{T}, g::AbstractGraph{V}) = @graph_requires g edge_map
 
 type AttributeEdgePropertyInspector{T} <: AbstractEdgePropertyInspector{T}
-  attribute::UTF8String
+  attribute::String
```
Comment (tkelman) [inline src/common.jl]: also Compat.UTF8String

**Comment by ViralBShah** [comment]
@tkelman Should we move Graphs.jl to a new org and then tag a new release?

**Comment by tkelman** [comment]
which org? JuliaArchive or something else?

**Comment by ViralBShah** [comment]
JuliaArchive sounds good.

**Comment by ViralBShah** [comment]
Should we also rename it to LegacyGraphs in the process?

**Comment by tkelman** [comment]
Renaming basically means re-registering a new copy from scratch. As described in https://github.com/JuliaLang/Graphs.jl/issues/227#issuecomment-242302832, it's not worth doing that unless we do a merge commit and replace the content with something new.

**Comment by ViralBShah** [comment]
Ok, let's just move it.

**Comment by tkelman** [comment]
Moved, but we should document the fact that this isn't actively maintained. https://github.com/JuliaArchive/Graphs.jl/issues/227#issuecomment-242294626

**Comment by ViralBShah** [comment]
Can you add a note to the README?

**Comment by ViralBShah** [comment]
Although isn't being in JuliaArchives enough of a signal?

**Comment by tkelman** [comment]
JuliaArchive doesn't have any description of what it means for a package to be here. You also only briefly see what user/organization a package comes from when you do `Pkg.add`.

**Comment by ViralBShah** [comment]
Is README enough?

@ranjanan Can you also register the new version in METADATA?

**Comment by tkelman** [comment]
Remember to do `Pkg.tag("Graphs", :minor)` since support for Julia 0.3 was dropped.

**Comment by ranjanan** [comment]
Sure, will tag a new version once #229 is merged.

**Comment by ranjanan** [comment]
Tagged. (https://github.com/JuliaLang/METADATA.jl/pull/6148)

**Pull request merged**